README.md

```diff
@@ -1,180 +1,75 @@
-# Swift-DocC
-
+<h1 align="center">
+    Swift-DocC
+    <br>
+    <br>
+    <a href="https://www.swift.org/documentation/docc/">
+        <img src="https://www.swift.org/images/docc-hero@2x.png" alt="DocC" width="500">
+    </a>
+    <br>
+</h1>
+
+<h4 align="center">
 Swift-DocC is a documentation compiler for Swift frameworks and packages aimed 
 at making it easy to write and publish great developer documentation.
+</h4>
```

Comment:
I think we should remove this additional formatting for now. We can consider it as part of its own PR – but for now, I'd rather we stay consistent with the rest of the README's in the Swift project.


```suggestion
# Swift-DocC

Swift-DocC is a documentation compiler for Swift frameworks and packages aimed 
at making it easy to write and publish great developer documentation.
```

Reply:
Ok!